cinema/serializers.py

```diff
@@ -63,12 +88,14 @@ class Meta:
 class MovieSessionListSerializer(MovieSessionSerializer):
     movie_title = serializers.CharField(source="movie.title", read_only=True)
     cinema_hall_name = serializers.CharField(
-        source="cinema_hall.name", read_only=True
+        source="cinema_hall.name",
+        read_only=True
     )
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
According to the checklist, you should not use `serializers.CharField` for adding an image. Instead, use `serializers.ImageField`. In this case, you have correctly used `serializers.ImageField` for `movie_image`, so no changes are needed here.